### PR TITLE
Create the first file fork_test

### PR DESCRIPTION
**Pull request by yhkim8505**
Created the for_test file.

Owner: Youngho Kim

I have added fork_test file for the following reasons.
1) Test fork in Github.
2) Test code review in Github.

### CODE REVIEW

File: fork_test [hunk 1]
```diff
@@ -0,0 +1 @@
+fork test
```
Comment (Inha-TCLab) [inline fork_test:1]: Can you add more explanations about the functionality of this part?

Comment (yhkim8505) [reply fork_test:1]: Okay.

**Review by yhkim8505 — COMMENTED** [review]
I done your comment. Thank you.

**Review by Inha-TCLab — APPROVED** [review]

**Comment by Inha-TCLab** [comment]
This is a create commit, but hope to recommend a few things. Please see the file to find out details.

**Pull request merged**